packages/quantic/force-app/main/default/lwc/quanticSourceCitations/__tests__/quanticSourceCitations.test.js

```diff
@@ -0,0 +1,196 @@
+      'How To Resolve Netflix Android Connection Error on XB, XBR, and XBR6 Smart TV',
+    uri: 'https://coveodemo.atlassian.net/wiki/TV/How To Resolve Netflix Android Connection Error on XB, XBR, and XBR6 Smart TV',
+    clickUri:
+      'https://coveodemo.atlassian.net/wiki/spaces/TV/pages/2359300/How+To+Resolve+Netflix+Android+Connection+Error+on+XB%2C+XBR%2C+and+XBR6+Smart+TV',
+    permanentid: '681eb97870fc99957bbd47e4c96d35bd88be7a6938f0797ee75991e4ab2a',
+  },
+  {
+    id: '302ede92620f05146aa97d10eb07a9eba7a7baec23434214bd7c8887617a-e024eeb5-4bdc-46d6-9d22-36cf9676871a',
+    title: 'How To Resolve Netflix Playback Errors on Besttech XB Smart TVs',
+    uri: 'https://coveodemo.atlassian.net/wiki/TV/How To Resolve Netflix Playback Errors on Besttech XB Smart TVs',
+    clickUri:
+      'https://coveodemo.atlassian.net/wiki/spaces/TV/pages/2359298/How+To+Resolve+Netflix+Playback+Errors+on+Besttech+XB+Smart+TVs',
+    permanentid: '302ede92620f05146aa97d10eb07a9eba7a7baec23434214bd7c8887617a',
+  },
+  {
+    id: '302ede92620f05146aa97d10eb07a9eba7a7baec23434214bd7c8887617a-952c4fec-d9a1-47ba-aa4c-1ea3d132c841',
+    title: 'How To Resolve Netflix Playback Errors on Besttech XB Smart TVs',
+    uri: 'https://coveodemo.atlassian.net/wiki/TV/How To Resolve Netflix Playback Errors on Besttech XB Smart TVs',
+    clickUri:
+      'https://coveodemo.atlassian.net/wiki/spaces/TV/pages/2359298/How+To+Resolve+Netflix+Playback+Errors+on+Besttech+XB+Smart+TVs',
+    permanentid: '302ede92620f05146aa97d10eb07a9eba7a7baec23434214bd7c8887617a',
+  },
+];
+
+const defaultOptions = {
+  citations: mockCitations,
+  citationClickHandler: functionsMocks.mockCitationClickHandler,
+};
+
+const selectors = {
+  citation: '.citation',
+  citationIndex: '.citation__index',
+};
+
+function createTestComponent(options = defaultOptions) {
+  const element = createElement('c-quantic-source-citations', {
+    is: QuanticSourceCitations,
+  });
+
+  for (const [key, value] of Object.entries(options)) {
+    element[key] = value;
+  }
+
+  document.body.appendChild(element);
+  return element;
+}
+
+// Helper function to wait until the microtask queue is empty.
+function flushPromises() {
+  return new Promise((resolve) => setTimeout(resolve, 0));
+}
+
+describe('c-quantic-source-citations', () => {
+  function cleanup() {
+    // The jsdom instance is shared across test cases in a single file so reset the DOM
+    while (document.body.firstChild) {
+      document.body.removeChild(document.body.firstChild);
+    }
+    jest.clearAllMocks();
+  }
+
+  afterEach(() => {
+    cleanup();
+  });
+
+  describe('when there are citations found', () => {
+    it('should properly display the citations', async () => {
+      const element = createTestComponent();
+      await flushPromises();
+
+      const citations = element.shadowRoot.querySelectorAll(selectors.citation);
+      expect(citations).not.toBeNull();
+      expect(citations.length).toBeGreaterThan(0);
```

Comment:
This line is unnecessary since in the following line you are comparing `citations.length` with `mockCitations.length`